bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/wrappers/BigtableHBaseSettings.java

```diff
@@ -39,6 +39,12 @@ public abstract class BigtableHBaseSettings {
   private final String instanceId;
   private final int ttlSecondsForBackup;
 
+  public static final String BULK_MUTATION_CLOSE_TIMEOUT_MILLISECONDS =
```

Comment:
Can we add `@VisibleForTesting` and a comment on "this is used to override the default closing timeout of 10 minutes in the test so the test can run faster".